deployment/README.md

```diff
@@ -1 +1,47 @@
 # Deployment
+
+## Requirements
+
+* Helm
```

Comment:
Any version in particular? If any, I guess it's safer to use your current version, so we have a reference in case in the future stop working.

Reply:
well minimum helm 3

I have this one: version.BuildInfo{Version:"v3.4.0", GitCommit:"7090a89efc8a18f3d8178bf47d2462450349a004", GitTreeState:"clean", GoVersion:"go1.14.10"}

Reply:
Can you update the document to state this?